### PR TITLE
pagination fix

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: engine/src/main/java/org/entando/entando/web/common/model/PagedMetadata.java [hunk 1]
```diff
@@ -126,4 +134,25 @@ public void setFilters(Filter[] filters) {
...
+    public void setActualSize(int actualSize) {
+        this.actualSize = actualSize;
+    }
+
+    public void imposeLimits() {
+        if (((this.getPage() - 1) * this.getActualSize()) > this.getBody().size()) {
+            this.setBody(new ArrayList<>());
+        } else {
+            int start = ((this.getPage() - 1) * this.getActualSize());
+            int end = (this.getPage() * this.getActualSize()) <= this.getBody().size() ? (this.getPage() * this.getActualSize())
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/common/model/PagedMetadata.java]: This is way way too complex. I can't follow what is happening at all. Break this up into multiple lines with variable names that tell us what is happening. Avoid the tertiary operator in chains.

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/web/common/model/PagedMetadata.java (hunk 1 above)
Comment (paddeo) [reply engine/src/main/java/org/entando/entando/web/common/model/PagedMetadata.java]: I deleted a useless line (now we don't have tertiary operator in chains) and added a comment.

**Pull request merged**